tool/clusterfuzz/commands/reproduce.py

```
@@ -59,6 +59,7 @@ def __enter__(self):
   def __exit__(self, unused_type, unused_value, unused_traceback):
     os.dup2(self.stdout, 1)
     os.dup2(self.stderr, 2)
+    return True
```

Returning true ignores the error.